streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -122,15 +122,13 @@
 
     /** {@code connections.max.idle.ms} */
     public static final String CONNECTIONS_MAX_IDLE_MS_CONFIG = CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_CONFIG;
-    private static final String CONNECTIONS_MAX_IDLE_MS_DOC = CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC;
```

Comment:
We can use `CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC` directly -- it's confusing to introduce a new name for the same thing.